libethcore/EVMSchedule.h

```diff
@@ -156,6 +158,9 @@ static const EVMSchedule IstanbulSchedule = [] {
     schedule.extcodehashGas = 700;
     schedule.haveChainID = true;
     schedule.haveSelfbalance = true;
+    schedule.eip1283Mode = true;
```

Comment:
Do we still need this var? eip1283 was rejected from Istanbul in favor of eip2200: https://eips.ethereum.org/EIPS/eip-1679

Can we control both the sstore and sload net gas metering via eip2200Mode?

Reply:
Well it's basically the same rules... 
(These two EIPs differ only in some price constants and in additional "throw below 2300 gas" rule)

But yeah, I think it would make sense to have `eip1283Mode = true` only in Constantinople, and then I'll change  `sstoreNetGasMetering()` to return `true` for both `eip1283Mode` and `eip2200Mode`

Reply:
changed